### PR TITLE
Open the UDP ports for a working Host provisioning.

### PR DESCRIPTION
**Pull request by kbidarkar**
a) Opened the UDP port 53 for DNS Capsule Feature
b) Opened the TFTP port 69 for TFTP Capsule Feature

### CODE REVIEW

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -376,11 +376,18 @@ def setup_firewall():
         5647,
         # Port 8000 for foreman-proxy service
         8000,
-        # Port 8443 for Katello access the Islated Capsule
+        # Port 8443 for Katello access the Isolated Capsule
```
Comment (elyezer) [inline automation_tools/__init__.py]: :+1:

File: automation_tools/__init__.py [hunk 2]
```diff
@@ -353,7 +353,7 @@ def setup_fake_manifest_certificate(certificate_url=None):
 
 def setup_firewall():
     """Setup firewall rules that Satellite 6 needs to work properly"""
-    ports = (
+    ports_tcp = (
```
Comment (elyezer) [inline automation_tools/__init__.py]: s/ports_tcp/tcp_ports?

Not a blocker

File: automation_tools/__init__.py [hunk 3]
```diff
@@ -376,11 +376,18 @@ def setup_firewall():
         5647,
         # Port 8000 for foreman-proxy service
         8000,
-        # Port 8443 for Katello access the Islated Capsule
+        # Port 8443 for Katello access the Isolated Capsule
         8443,
     )
 
-    for port in ports:
+    ports_udp = (
```
Comment (elyezer) [inline automation_tools/__init__.py]: s/ports_udp/udp_ports?

Not a blocker

**Comment by elyezer** [comment]
We have some code duplication going on, what about:

```python
definitions = {
    'tcp': (
        1234,
        3456,
    )
    'udp': (
        8765,
        4321,
    )
}

for protocol, ports in definitions:
    for port in ports:
         rule_exists = run(
              r'iptables -nL INPUT | grep -E "^ACCEPT\s+{0}.*{1}"'.format(protocol, port),
              quiet=True,
         ).succeeded
         if not rule_exists:
             run(
                 'iptables -I INPUT -m state --state NEW -p {0} --dport {1} '
                  '-j ACCEPT'.format(protocol, port)
              )
```

**Comment by elyezer** [comment]
ACK, pending comment.

**Comment by elyezer** [comment]
ACK

**Comment by sthirugn** [comment]
ACK

**Comment by rplevka** [comment]
this code does not work:
```python
    return self.wrapped(*args, **kwargs)
  File "/path/to/automation_tools/__init__.py", line 395, in setup_firewall
    for protocol, ports in definitions:
ValueError: too many values to unpack
```

**Pull request merged**